src/main/cpp/http/prototype.cpp

```diff
@@ -0,0 +1,111 @@
+#include "prototype.h"
+#include <iostream>
+
+using namespace std;
+using namespace Stride;
+
+
+// HelloWorldRequestHandler
+
+HelloWorldRequestHandler::HelloWorldRequestHandler(const std::string& format): _format(format) {}
+
+void HelloWorldRequestHandler::handleRequest(HTTPServerRequest& request, HTTPServerResponse& response){
+    Application& app = Application::instance();
+    app.logger().information("Request from "
+        + request.clientAddress().toString());
```

Comment:
(Ehh, if you can’t get `clang-format` to work, it’s not a big deal, we can always format stuff later :))